processors/config/cnvStdhepToRoot_cfg.py

```diff
@@ -29,7 +31,7 @@
 #MCParticles
 cnvStd.parameters["mcPartCollStdhep"] = 'MCParticle'
 cnvStd.parameters["mcPartCollRoot"] = 'MCParticle'
-cnvStd.parameters["maxEvent"] = options.nevents
+cnvStd.parameters["maxEvent"] = options.skip_events+options.nevents
```

Comment:
This won't work if options.nevents is -1, which is pretty typical.

Reply:
I had to do change three files to get a consistent thing. Let me know if you think I've done it right.